.github/workflows/homebrew.yml

```diff
@@ -0,0 +1,39 @@
+name: "Smoke Test Homebrew install"
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request: # TODO: remove pull_request trigger
+
+jobs:
+  smoke-test-homebrew:
+    name: Test on ${{ matrix.runner }}
+    runs-on: ${{ matrix.runner}}
+    strategy:
+      matrix:
+        runner: [macos-13, macos-14, macos-15]
+      fail-fast: false
+    steps:
+      - run: brew install mongosh
+      - run: mongosh --smokeTests
+      - name: Report failure
+        # TODO: replace with failure()
+        if: ${{ success() }}
```

Comment:
Open TODO?

Reply:
Yeah, I'll fix that before we merge - just wanted to test it first once I get the slack webhook from IT.